addons/field_rations/stringtable.xml

```diff
@@ -138,77 +139,89 @@
         </Key>
         <Key ID="STR_ACEX_Field_Rations_ThirstQuenched_DisplayName">
             <English>Thirst Quenched</English>
+            <German>Durst gestillt</German>
             <Chinese>止渴程度</Chinese>
             <Japanese>乾きの満たし</Japanese>
             <Russian>Утоление жажды</Russian>
         </Key>
         <Key ID="STR_ACEX_Field_Rations_ThirstQuenched_Description">
             <English>Coefficient for the amount of thirst quenched from drinking</English>
+            <German>Koeffizient für die Menge des Durst stillen Werts vom Trinken</German>
             <Chinese>從飲用中能夠攝取多少水份。</Chinese>
             <Japanese>飲むことによって得られる乾きの満たし量の係数を設定します</Japanese>
             <Russian>Коэффициент утоления жажды от питья</Russian>
         </Key>
         <Key ID="STR_ACEX_Field_Rations_HungerSatiated_DisplayName">
             <English>Hunger Satiated</English>
+            <German>Hunger gesättigt</German>
             <Chinese>飽足感</Chinese>
             <Japanese>満腹感</Japanese>
             <Russian>Утоление голода</Russian>
         </Key>
         <Key ID="STR_ACEX_Field_Rations_HungerSatiated_Description">
             <English>Coefficient for the amount of hunger satiated from eating</English>
+            <German>Koeffizient für die Menge des Hunger gesättigt Werts vom Essen</German>
             <Chinese>從飲食中獲取多少飽食感。</Chinese>
             <Japanese>食べることによって得られる満腹感の量の係数を設定します</Japanese>
             <Russian>Коэффициент утоления голода от принятия пищи</Russian>
         </Key>
         <Key ID="STR_ACEX_Field_Rations_AffectAdvancedFatigue_DisplayName">
             <English>Affect Advanced Fatigue</English>
+            <German>Beeinflusst die Erweiterte Ausdauer</German>
             <Chinese>與進階疲勞聯動</Chinese>
             <Japanese>アドバンスド ファティーグへの影響</Japanese>
             <Russian>Влияние на Усталость</Russian>
         </Key>
         <Key ID="STR_ACEX_Field_Rations_AffectAdvancedFatigue_Description">
             <English>Controls if thirst and hunger should affect ACE Advanced Fatigue</English>
+            <German>Bestimmt ob Durst und Hunger ACE Erweiterte Ausdauer beeinflussen sollen.</German>
```

Comment:
Bestimmt,

Reply:
erweiterte ist ein Adjektiv?
Unless we take it as a "Name" here? Not sure